clients/nexus-lockstep-client/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Interface for making API requests to the Oxide control plane at large from
+//! callers that update in lockstep with Nexus itself (e.g. rack initialization,
+//! tests and debugging)
```

Comment:
Can you file an issue for "have ls-apis make sure that we don't ship anything that uses this API"?  I expect we'll have to make at least RSS an exception and maybe omdb too, but I think it'd be worth doing it.

Reply:
https://github.com/oxidecomputer/omicron/issues/9057